.github/workflows/clear_self_hosted_persistent_caches.yaml

```diff
@@ -7,8 +7,10 @@ jobs:
   clean_linux_arm64:
     runs-on:
     - self-hosted
-    - Linux
-    - ARM64
+    - runs-on
+    - runner=4cpu-linux-arm64
+    - image=ubuntu22-full-arm64-python3.7-python3.8-python3.9
```

Comment:
This points to a custom AMI that has those pythons.

Reply:
Are there instructions on how to build the AMI? Or better yet, is there a Packer config to build the AMI programmatically?

Reply:
There is no packer config, but I will write up how to build the AMI manually (it's very simple). If someone wants to turn that into a packer config, great, but I didn't need more learning curve on this one...

Reply:
@benjyw Great to see that you managed to merge this! Note that you will need to rebuild the image quite regularly (every 15 to 30 days is best), because at some point the GitHub runner agent binary will refuse to start if it's stuck on a too old version.

Reply:
Thanks for the heads up, we will have to cross that bridge when we come to it. I assume we'll want to automate this.